README.md

```diff
@@ -61,6 +61,8 @@ use: "file-loader?name=[name].[ext]&publicPath=assets/foo/&outputPath=app/images
 }
 ```
 
+`cssOutputPath` also should be used (together with `useRelativePath` property) to define the output path context.
```

Comment:
Please add e.g an example for this awell :). The current explanation is too jaded to reason about it (especially for beginners)